### DISCUSSION TITLE
KERI Implementer's Call 11/05/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#103] [Meetings]
Zoom:
https://us06web.zoom.us/j/84721071832

Reports:
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

**Comment by pfeairheller** [comment]
## SignifyPy

* No updates

**Comment by lenkan** [comment]
Can we quickly discuss this: https://github.com/WebOfTrust/keri/pull/102 (Draft: workflow for creating issue report)

**Reply by lenkan** [reply]
I just ran it on my fork and it produced this summary: https://github.com/lenkan/keri/discussions/9

**Comment by iFergal** [comment]
Still stuck on this OOBI discovery topic haha

If there's time, I'd like to discuss/ask how to include an external service endpoint that's just used for .well-known OOBI discovery. Different domain/subdomain to KERIA.

**Comment by rodolfomiranda** [comment]
## KERIA:
### PRs:
- https://github.com/WebOfTrust/keria/pull/313
- https://github.com/WebOfTrust/keria/pull/314
- https://github.com/WebOfTrust/keria/pull/315

### Issues:
- https://github.com/WebOfTrust/keria/issues/312

**Reply by 2byrds** [reply]
Have @rodolfomiranda and/or @iFergal and/or @lenkan been given PR review capabilities? They are very active in this repo and it would help for all three of them to be able to move the reviews forward.

**Reply by daviddm** [reply]
Just created a new issue https://github.com/WebOfTrust/keria/issues/316

**Comment by 2byrds** [comment]
## Signify-browser-extension:
Submission process to the chrome catalog has begun

**Comment by 2byrds** [comment]
## Signify-ts:
### PRs:
- https://github.com/WebOfTrust/signify-ts/pull/286 vs. https://github.com/WebOfTrust/signify-ts/pull/271
- https://github.com/WebOfTrust/signify-ts/pull/282

**Comment by m00sey** [comment]
keripy:

**prs**
- https://github.com/WebOfTrust/keripy/pull/867
- https://github.com/WebOfTrust/keripy/pull/868
- https://github.com/WebOfTrust/keripy/pull/878
- https://github.com/WebOfTrust/keripy/pull/882

**issues**
- https://github.com/WebOfTrust/keripy/issues/885
- https://github.com/WebOfTrust/keripy/issues/884
- https://github.com/WebOfTrust/keripy/issues/883

**Reply by iFergal** [reply]
I also opened this issue which Sam said we should put on the agenda: https://github.com/WebOfTrust/keripy/issues/884

(for me higher priority than my other discussion item :P)

**Reply by m00sey** [reply]
started looking at the decoupling will require changes for GLEIFs usage so is essentially a breaking change for us. the kli replacement is irrelevant we're moving away from it.

**Reply by iFergal** [reply]
@m00sey @kentbull Do you guys think you'll move towards taking on this change at some point? For now, I can at least change to `.update` from `.replace` to avoid the issue of overwriting on subsequent resolutions which should be non breaking?

**Reply by kentbull** [reply]
Yes, we should change to .update from .replace.
